images/linux/toolsets/toolset-2004.json

```diff
@@ -103,7 +103,11 @@
     "azureModules": [
         {
             "name": "az",
+            "url" : "https://raw.githubusercontent.com/Azure/az-ps-module-versions/main/versions-manifest.json",
             "versions": [
+                "5.5.0"
+            ],
+            "zip_versions": [
                 "3.8.0",
```

Comment:
Please check if these versions are needed. Most used version in ubuntu is 3.1.0, most likely that would be better to bundle instead of these versions.
@maxim-lobanov can u confirm if these versions are really needed in Ubuntu 2004

Reply:
ubuntu 2004 was always released with these versions. if we don't include this by default unnecessarily we are increasing time for existing pipelines who are using these versions. 

Also there is provision now to download any specific version who ever wants this version they can use it.

I don't think we should change existing behaviour

Reply:
@AmrutaKawade the reason for my ask is because we have a plan that `ubuntu-latest` will start making use of 2004 instead of 1804, so many jobs will switch or would have already switched to start making use of 2004 from 1804. These jobs will get affected as the az versions would then get dynamically downloaded.
If you look at the telemetry data as of today:
- 3.8.0 usage in ubuntu is 54
- 4.8.0 usage in ubuntu is 1497
whereas
- 3.1.0 usage in ubuntu is 18746 out of which 15997 is happening through GHRelease

Reply:
Below is the telemetry data sorted based on linuxJobCount:
![image](https://user-images.githubusercontent.com/8944724/112264048-4bd27980-8c96-11eb-857e-9a82d5763250.png)